app/src/pages/inside/dashboardItemPage/dashboardItemPage.jsx

```diff
@@ -230,19 +239,35 @@ export class DashboardItemPage extends Component {
           <div className={cx('dashboard-item')}>
             <div className={cx('buttons-container')}>
               <div className={cx('nav-left')}>
-                <GhostButton icon={AddWidgetIcon} onClick={this.showWidgetWizard}>
-                  {formatMessage(messages.addNewWidget)}
-                </GhostButton>
-                <GhostButton icon={AddSharedWidgetIcon} onClick={this.showAddSharedWidgetModal}>
-                  {formatMessage(messages.addSharedWidget)}
-                </GhostButton>
+                {!isOwner && (
```

Comment:
Can we write it as: `isOwner ? (block with caption) : (<Fragment>buttons</Fragment>)`